H/HiGHS/build_tarballs.jl

```diff
@@ -7,7 +7,7 @@ name = "HiGHS"
 version = v"0.2.3"
 
 sources = [
-    GitSource("https://github.com/ERGO-Code/HiGHS.git", "289e34dc089dda8260e93eebd09e724ef4c64ac3"),
+    GitSource("https://github.com/ERGO-Code/HiGHS.git", "73ec415532691dbb4270d4c1952567b0f779a8aa"),
```

Comment:
Should the version number change since the source changed?  I'm actually not sure what versioning scheme we're following here :thinking:

Reply:
Upstream isn't versioning their code pre-1.0. So I've been using `0.x.y` freely.